docs/windows_dev.md

```diff
@@ -0,0 +1,99 @@
+# Integration Tests: Windows Local setup
+**Prerequisites**: 
+- You have already setup Go and the buildkit repo and you can build from source.
+
+
+### Main Steps
+- Install registry.exe:
+```go install -v github.com/distribution/distribution/v3/cmd/registry@latest```
+- Install buildkit with the following command: 
+     ```
+    cd to repo directory
+    go install -v .\cmd\...
+    ```
+
+- Run buildkitd from the _$env:PATH_. Also remember to remove any other buildkitd installed in $env:ProgramFiles.
```

Comment:
Remove any other buildkitd installed in $env:ProgramFiles.

Reply:
Actually there's no need to remove the buildkit EXEs but rather just remove them from `$env:PATH`; so that they are not picked up by the integration test setup when spinning off the buildkitd sandbox.